comment by tim

cty288 opened the pull request.

b-chen00 commented:
Hello, can you add back the visual changes made to my paragraph? I used <div align="right"">-Kavindu Dodanduwa</div> and <div align="right"">Brandon Chen</div> to improved the visual rather than just blockquotes. Thank you!

cty288 commented:
> Hello, can you add back the visual changes made to my paragraph? I used `<div align="right"">-Kavindu Dodanduwa</div>` and `<div align="right"">Brandon Chen</div>` to improved the visual rather than just blockquotes. Thank you!

I just reverted and added that back. See if it's all right on your side

b-chen00 commented:
Looks good, working on merging right now.
